modules/administration/pages/tshoot-registerclones.adoc

```diff
@@ -78,7 +78,7 @@ or:
 . Stop osad with:
 +
 ----
-# /etc/init.d/osad stop
+# /etc/init.d/mgr-osad stop
```

Comment:
Two notes about this:

1. The service is still `osad`.
2. On systemd services (SLE12SP3, for example), this file does not exist, and instead osad should be stopped with `systemctl stop osad` or `service mgr-osad stop`